types/MimicryMacroMarketMashup.md

```diff
@@ -0,0 +1,211 @@
+## Type Name
+
+- `MimicryMacroMarketMashup`
+
+
+## Description
+
+This query returns a single market capitalization metric derived from the sum of zero or more NFT collection market capitialization metrics, plus zero or more ERC-20 market capitalization metrics.
+
+Its initial use case is enabling the creation of macro-markets on [Mimicry Protocol](https://www.mimicry.org/), but the values can be used by anyone who needs this metric.
+
+### Market Capitalization for NFT Collections, Explained
+Market capitalization is calculated as the sum of each NFT within an NFT collection, where each NFT is valued at the greater of its last traded price or the floor price of the collection, respectively.
+
+#### For ERC721 collections:
+- Market Cap = the sum of all NFT values of the collection
+- The value of each NFT = max(last price, floor price)
+
+#### For ERC1155 collections:
+- Market Cap = the sum of all NFT values of the collection
+- The value of each NFT = max(last price, floor price) times the number of owners
+
+Suspected wash trades should be filtered out.
```

Comment:
is this just off of nftGo's spec?  Or just a best attempt?

Reply:
This came straight from NFTGo. Re: https://docs.nftgo.io/docs/collection-metrics